client/my-sites/plugins/plugin-information/index.jsx

```diff
@@ -180,10 +179,6 @@ class PluginInformation extends React.Component {
 	};
 
 	getActionLinks = ( plugin ) => {
-		if ( this.props.calypsoify ) {
```

Comment:
Removed Calypsoify reference to show action links for plugins on Atomic sites:
<img width="733" alt="Screen Shot 2021-06-29 at 3 34 36 PM" src="https://user-images.githubusercontent.com/1689238/123856873-9a279200-d8ef-11eb-91b0-b3585d0a426e.png">

Reply:
Because of the redirect, I'm struggling to see the result of this change. It looks fine though. We shouldn't be using Calypsoify at all in the plugins section anymore.